src/System.Private.CoreLib/shared/System/Buffer.cs

```diff
@@ -158,7 +160,7 @@ internal static unsafe void Memmove(byte* dest, byte* src, nuint len)
             // Copy bytes which are multiples of 16 and leave the remainder for MCPY01 to handle.
             Debug.Assert(len > 16 && len <= 64);
 #if HAS_CUSTOM_BLOCKS
```

Comment:
Under what scenarios is `HAS_CUSTOM_BLOCKS` false today? Only for the OOB package?

Reply:
```csharp
#if AMD64 || ARM64 || (BIT32 && !ARM)
#define HAS_CUSTOM_BLOCKS
#endif
```

Reply:
Don't think there is an OOB package?

Reply:
It might be worth just testing this again on 32-bit ARM (maybe not now, but post 3.0) to see if this is any better and the `#if` blocks can just be removed.